internal/handlers/telegram/handler_test.go

```diff
@@ -10,6 +10,160 @@ import (
 	"testing"
 )
 
+func MockUpdateHandler(updates tgbotapi.UpdatesChannel) string {
+	for u := range updates {
+		switch {
+		case u.Message == nil:
+			return "Missing message data"
+		case u.Message.NewChatMembers != nil:
+			return "joinHandler triggered"
+		case u.Message.LeftChatMember != nil:
+			return "partHandler triggered"
+		case u.Message.Text != "":
+			return "messageHandler triggered"
+		case u.Message.Sticker != nil:
+			return "stickerHandler triggered"
+		case u.Message.Document != nil:
+			return "documentHandler triggered"
+		case u.Message.Photo != nil:
+			return "photoHandler triggered"
+		default:
+			return "Triggered, but message type is currently unsupported"
+		}
+	}
+	return ""
+}
+
```

Comment:
I had to create this function here based on the handler itself, and remove the calls to the other methods to make it work.

Reply:
This looks incredibly fragile and prone to code rot if people forget this happened...

Reply:
@robbyoconnor Could you please clarify what you mean by if people forget this happens? This is in the context of unit tests, right?

Reply:
I withdraw my comment.